Grammar/python.gram

```diff
@@ -413,6 +415,11 @@ except_block[excepthandler_ty]:
         _PyAST_ExceptHandler(e, (t) ? ((expr_ty) t)->v.Name.id : NULL, b, EXTRA) }
     | 'except' ':' b=block { _PyAST_ExceptHandler(NULL, NULL, b, EXTRA) }
     | invalid_except_stmt
+except_star_block[excepthandler_ty]:
+    | invalid_except_star_stmt_indent
```

Comment:
I wonder if the "invalid" rules shouldn't be shared between `except` and `except*`. I also suspect that we may want to emit a custom error when users mix `except` and `except*`.

Reply:
A related comment: I think this check that "except:" is the last clause can move to the parser: https://github.com/python/cpython/blob/15409c720be0503131713e3d3abc1acd0da07378/Python/compile.c#L3251

But I don't think we should make changes to except (non-star) in this PR.

(note that we do not allow "except*" at all).

Reply:
I've updated the parser as suggested and created bpo45834 for moving the 'except:' error to the parser.